app/src/main/java/com/sopt/peekabookaos/presentation/bookshelf/BookshelfFragment.kt

```diff
@@ -32,13 +35,45 @@ class BookshelfFragment : BindingFragment<FragmentBookshelfBinding>(R.layout.fra
         binding.rvBookshelfPick.adapter = pickAdapter
         pickAdapter.submitList(viewModel.pickData.value)
 
-        friendAdapter = BookShelfFriendAdapter()
+        friendAdapter = BookShelfFriendAdapter { pos, item ->
```

Comment:
```suggestion
        friendAdapter = BookShelfFriendAdapter { pos, _ ->
```
사용하지 않는 람다식은 _ 처리 해주세요.